tests/e2e_logging.py

```diff
@@ -799,6 +875,9 @@ def run(args):
         network = test_network_node_info(network, args)
         network = test_metrics(network, args)
         network = test_memory(network, args)
+        if args.package == "liblogging":
```

Comment:
What we save in setup time, we spend again testing in bft, which we didn't before. Still a win.